src/index.html

```diff
@@ -1,12 +1,306 @@
 <!doctype html>
 <html lang="en">
+
   <head>
     <meta charset="UTF-8">
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
     <title>Catalog</title>
     <link rel="stylesheet" href="./styles/main.scss">
+    <link rel="preconnect" href="https://fonts.gstatic.com">
+    <link href="https://fonts.googleapis.com/css2?family=Roboto:wght@400;500;700&display=swap" rel="stylesheet">
   </head>
+
   <body>
-    <h1>Catalog</h1>
+    <header class="header">
+      <a href="#home" class="home">
+        <img src="images/logo.png" alt="Moyo logo">
+      </a>
+      <nav class="nav">
+        <ul class="nav__ul">
```

Comment:
```suggestion
        <ul class="nav__list">
```
Don't name classes according to the tags. Give meaningful names